planet_sheet/trade.tex

```diff
@@ -24,7 +24,7 @@
   - &
   \multicolumn{2}{l}{:) SSS T} &
    &
-  {\color{supplemental} S} &
```

Comment:
With the opponents reward being grayed out it seemed to be more like hint than actual things to do.